src/common/utils/gpo.h

```diff
@@ -305,4 +380,51 @@ namespace powertoys_gpo {
         return getConfiguredValue(POLICY_ALLOW_EXPERIMENTATION);
     }
 
+    inline gpo_rule_configured_t getRunPluginEnabledValue(std::string pluginID)
+    {     
+        if (pluginID == "" || pluginID == " ")
+        {
+            // this plugin id can't exist in the registry
+            return gpo_rule_configured_not_configured;
+        }
+
+        std::wstring plugin_id(pluginID.begin(), pluginID.end());
+        auto individual_plugin_setting = getPolicyListValue(POWER_LAUNCHER_INDIVIDUAL_PLUGIN_ENABLED_LIST_PATH, plugin_id);
+        
+        if (individual_plugin_setting.has_value())
```

Comment:
@jaimecbernardo
I can read the registry without fixed array size now. But I can't compare this `std::wstring`. Can you please help.